src/handlers/authenticationHandlers.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+
+import { pathObj } from './../utils/generalVariables';
+
+export const loginHandler = ({ username, password }) => {
+  // With the finalization of the database schema, more checks can be implemented (with separate error-messages)
+  if (!username || !password || typeof username !== "string" || typeof password !== "string" || password.length < 8 ) {
```

Comment:
Good idea for validating also in the frontend side. The backend is validating the password with a minimum of 6 characters, so we might want to do the same here https://github.com/tieme-ndo/backend/blob/master/middlewares/Validations/Users.js#L25

Reply:
Good call, will change this real quick!